apps/schematic/api/docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
     networks:
       - schematic
     ports:
-      - "${SERVER_PORT}:7080"
+      - "7443:7443"
```

Comment:
This change should be mirrored in `.devcontainer/devcontainer.json`

Reply:
Thanks for the reminder Thomas. I have updated devcontainer.json